src/__tests__/__snapshots__/union_strings.spec.js.snap

```diff
@@ -2,6 +2,7 @@
 
 exports[`should handle union strings 1`] = `
 "declare interface MyObj {
-    state?: 'APPROVED' | 'REQUEST_CHANGES' | 'COMMENT' | 'PENDING'
-}"
+  state?: \\"APPROVED\\" | \\"REQUEST_CHANGES\\" | \\"COMMENT\\" | \\"PENDING\\";
```

Comment:
Is this change correct?

Reply:
Yes, this is because prettier uses double quotes and the declare is surrounded with double quotes (see line 4)
I did an inspection to the test in debug, and the code looks fine:
```
declare interface MyObj {
  state?: "APPROVED" | "REQUEST_CHANGES" | "COMMENT" | "PENDING";
}
```